app/get_instances.gradle.kts

```diff
@@ -0,0 +1,103 @@
+import java.io.OutputStreamWriter
+import java.net.URL
+import java.net.HttpURLConnection
+
+// We can't import libraries here for some reason, so we must use what is provided
+// by gradle, which isn't much. The groovy JSON library is meant for use by groovy code,
+// so we need some creativity to use it in Kotlin.
+import org.apache.groovy.json.internal.LazyMap
+import groovy.json.JsonOutput
+import groovy.json.JsonSlurper
+
+val endpointUrl = "https://the-federation.info/v1/graphql"
```

Comment:
The-federation.info isn't that well maintained, and contains a lot of old and generally incorrect info, as evidenced by the errors in the list it generated below. 

Two other options:
- Use https://lemmy.fediverse.observer/ , which has an API. Sort by Active users in the last month, not total users.
- Use the https://github.com/LemmyNet/lemmy-stats-crawler/ , which generates a usable json file.

Reply:
I think the first option would be the most convenient, it would be pretty easy to modify the code to use that API instead.